corehq/apps/cloudcare/views.py

```diff
@@ -471,27 +473,35 @@ def put(self, request, *args, **kwargs):
 def report_formplayer_error(request, domain):
     data = json.loads(request.body)
     error_type = data.get('type')
+
+    with sentry_sdk.configure_scope() as scope:
+        scope.set_tag("cloudcare_error_type", error_type)
```

Comment:
Should this `with`-block contain all of the `notify_exception(...)` calls below? (I.E., should the code below this be indented on elevel?)

I'm finding it a little hard to tell exactly how scopes are expected to be used, but the part on [Local Scopes](https://docs.sentry.io/enriching-error-data/scopes/?platform=python#local-scopes) implies that this tag will not be sent with any of the "events" below.

Reply:
I was following the example here, which seems to be properly putting these into scope for the duration of the request: https://github.com/dimagi/commcare-hq/blob/b2bcde32cb50de4f78ec475d4182860c78d3a6bd/corehq/middleware.py#L210-L215.

The link you gave to Local Scopes uses a different method, `push_scope`, which appears to only affect sentry reports fired within that block, as opposed to `configure_scope()` which appears to persist for the whole "session"/request.